projects/storefrontlib/src/cms-components/product/carousel/product-references/product-references.component.service.spec.ts

```diff
@@ -113,6 +116,40 @@ describe('ProductReferencesService', () => {
     expect(title$).toBe(title);
   });
 
+  it('should have displayTitle', () => {
+    spyOn(productReferenceService, 'setDisplayProductTitles').and.callThrough();
+    spyOn(productReferenceService, 'getDisplayProductTitles').and.callThrough();
+
+    let displayTitle$: boolean;
+
+    productReferenceService.setDisplayProductTitles();
+    productReferenceService
+      .getDisplayProductTitles()
+      .subscribe(data => {
+        displayTitle$ = data;
```

Comment:
btw I think it looks cleaner is you: `.subscribe(data => (displayTitle$ = data))`